src/Menu/MenuBuilder.php

```diff
@@ -60,7 +60,10 @@ public function __construct(
         $this->pool = $pool;
         $this->factory = $factory;
         $this->provider = $provider;
-        $this->eventDispatcher = LegacyEventDispatcherProxy::decorate($eventDispatcher);
+
+        $legacyEventDispatcher = LegacyEventDispatcherProxy::decorate($eventDispatcher);
```

Comment:
Do we need this LegacyEventDispatcher anymore?

Reply:
Do you ask if it's (the event dispatcher) used? yes.
Do we need to decorate? No since `LegacyEventDispatcherProxy` is used to provide forward and/or backward compatibility with Symfony 3/4 packages. In our case we have symfony 3 dropped already